src/main/java/peoplesoft/logic/commands/job/JobDeleteCommand.java

```diff
@@ -22,10 +22,9 @@
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
             + ": Deletes the job identified by the index.\n"
-            + "Parameters: JOB_INDEX\n"
-            + "Example: " + COMMAND_WORD + " 1";
+            + "Example: " + COMMAND_WORD + " 3";
```

Comment:
Adding a "Format: JOB_INDEX" before "Example: " would probably be more consistent with the other commands